examples/lmdbxx/CMakeLists.txt

```diff
@@ -0,0 +1,14 @@
+cmake_minimum_required(VERSION 3.1)
+
+include("../common.cmake")
+
+project(download-lmdbxx)
+
+hunter_add_package(lmdbxx)
+
+find_package(lmdbxx CONFIG REQUIRED)
+
+set(CMAKE_CXX_STANDARD 11)
```

Comment:
Please remove, see:
* http://cgold.readthedocs.io/en/latest/tutorials/toolchain/globals/cxx-standard.html#summary

Reply:
I think this is going to affect the working builds. Will make another pull request to https://github.com/ingenue/hunter for pkg.lmdbxx after my build completes.

Reply:
https://github.com/ingenue/hunter/pull/185